src/frontend/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:lts-alpine as build
```

Comment:
Can we please make it a more specific version like `node:18.9-alpine` to avoid any unforeseen breaks in the future?

Reply:
That's a great point Kushal. Anthony and I ran into issues a month ago using this exact same image. 

Good catch.

Reply:
good point.